### PR TITLE
core_perception: 1.14.9-1 in 'melodic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by Timple**
Increasing version of package(s) in repository `core_perception` to `1.14.9-1`:

- upstream repository: https://github.com/nobleo/core_perception.git
- release repository: https://github.com/nobleo/core_perception-release.git
- distro file: `melodic/distribution.yaml`
- bloom version: `0.9.8`
- previous version for package: `null`

## points_preprocessor

```
* Remove Autoware Health Checker as dependency
* Contributors: Tim Clephas
```

### CODE REVIEW

**Comment by Timple** [comment]
I could not find any documentation on this package, so I'm leaving that out.

Releasing as discussed in https://github.com/Autoware-AI/core_perception/issues/7

**Review by mabelzhang — COMMENTED** [review]
Looks good to me.
New package - I'll wait for another reviewer to double-check and merge.
Also holding for [Melodic sync](https://discourse.ros.org/t/preparing-for-melodic-sync-2020-08-19/15988)

**Review by jacobperron — APPROVED** [review]

**Pull request merged**
